src/pages/lost-items/lost-items.tsx

```diff
@@ -1,5 +1,78 @@
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '@shared/components/button/button';
+import Card from '@shared/components/card/card';
+import { themeVars } from '@shared/styles';
+
+import { LOST_ITEMS } from './constants/lostItems';
+import * as styles from './lost-items.css';
+
+const lostMock = [
+  {
+    id: 1,
+    title: '100만원',
+    description: '길가에서 주운 100만원',
+    src: '',
+    alt: '',
+  },
+  {
+    id: 2,
+    title: '전공책',
+    description: '아리관 6층 3번째 계단',
+    src: '',
+    alt: '',
+  },
+];
+
 const LostItems = () => {
-  return <div>LostItems</div>;
+  const [isAdmin, setIsAdmin] = useState(false);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const userRole: 'user' | 'admin' = 'admin';
+    setIsAdmin(userRole === 'admin');
+  }, []);
+
+  const handleCardClick = (id: number) => {
+    navigate(`/lost-items/${id}`);
```

Comment:
```suggestion
    navigate(`${routePath.LOST_ITEMS}/${id}`);
```
이 부분도 routePath 사용하기!

Reply:
수정햇슴다